target/linux/ath79/dts/qca9531_glinet_gl-xe300-nor-nand.dts

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "glinet,gl-xe300", "qca,qca9531";
```

Comment:
@nunojpg So, why would you choose that compatible? Are there other variants available? If yes, everything should have the suffix, and I currently miss it on DEVICE_MODEL and model property?
Or can we just remove it since it's not relevant for discrimination between devices?

Reply:
-nor-nand was also part of the filename (qca9531_glinet_gl-xe300-nor-nand.dts) and "define Device/glinet_gl-xe300-nor-nand". So this was the off part. I agree that probably it is enough to rename the other two and leave this as it was.

Reply:
So this was just a drive-by comment and you don't have any specific knowledge either?
So, let's find somebody who knows more about the variants of this device before we change it backwards and forwards uselessly again.

Reply:
@adschm I know more about this device, the [GL-XE300](https://www.gl-inet.com/products/gl-xe300/) always has both the nor and nand flash included. I suggest that the best approach here is that used for the similar [gl-e750](https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9531_glinet_gl-e750.dts) which is named `qca9531_glinet_gl-e750.dts`, and doesn't include a -nor-nand suffix. This seems the best approach as the initial support will make use of the nand.

An example of a device making use of the -nor-nand suffix is the [GL-AR300M](https://www.gl-inet.com/products/gl-ar300m/) which also has both nor and nand flash included, apparently in all models. There are [nor](https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9531_glinet_gl-ar300m-nor.dts) and [nand](https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9531_glinet_gl-ar300m-nand.dts) builds for the GL-AR300M.

I don't know why this is as initial support in d963ddf does include the nand, perhaps OpenWrt didn't make use of the nand for the overlay at first?

Reply:
I just removed all '-nor-nand' suffixes.
Thank you guys for your remarks.